Sources/SwiftPhoenixClient/Socket.swift

```diff
@@ -530,10 +403,10 @@ public class Socket: PhoenixTransportDelegate {
     /// call this method when you are finished when the Socket in order to release
     /// any references held by the socket.
     public func releaseCallbacks() {
-        self.stateChangeCallbacks.open.removeAll()
-        self.stateChangeCallbacks.close.removeAll()
-        self.stateChangeCallbacks.error.removeAll()
-        self.stateChangeCallbacks.message.removeAll()
+        stateChangeCallbacks.open.removeAll()
+        stateChangeCallbacks.close.removeAll()
+        stateChangeCallbacks.error.removeAll()
+        stateChangeCallbacks.message.removeAll()
```

Comment:
Removing `self.` here and in various other locations above isn't directly related to removing `delegate*`, also the removal is inconsistent with other places that keep `self.`. This is a nit, but I prefer to keep `self.` where it has already been declared. Again, I prefer the explicitness at the cost of verbosity. Also, it reduced the change diff to be isolated purely to just the removal of `delegate*`